fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -765,7 +765,13 @@ protected void addCacheControlHeaders(final HttpServletResponse servletResponse,
         }
 
         if (!etag.getValue().isEmpty()) {
-            servletResponse.addHeader("ETag", etag.toString());
+            final String etagStrVal = etag.toString();
```

Comment:
Have you inspected the value of `etagStrVal`? It is a weak-etag, no? i.e. `W/123456`

Reply:
It is a weak etag.  Is that a problem? Any recommendations?

Reply:
I believe the only reason we return weak etags is due to the lack of guarantees around the order of RDF triples.
It is probably a non-issue. It may seem a little less hokey if we don't include the `W/` in our state-token... but not a big deal.